tests/models/test_admin.py

```diff
@@ -1,156 +1,180 @@
-        assert False == self.admin.is_librarian(self._default_library)
-
-        other_library = self._library()
-        self.admin.add_role(AdminRole.LIBRARY_MANAGER, other_library)
-        assert False == self.admin.is_library_manager(self._default_library)
-        assert True == self.admin.is_library_manager(other_library)
-        self.admin.add_role(AdminRole.SITEWIDE_LIBRARIAN)
-        assert False == self.admin.is_library_manager(self._default_library)
-        assert True == self.admin.is_library_manager(other_library)
-        assert True == self.admin.is_librarian(self._default_library)
-        assert True == self.admin.is_librarian(other_library)
-        self.admin.remove_role(AdminRole.LIBRARY_MANAGER, other_library)
-        assert False == self.admin.is_library_manager(self._default_library)
-        assert False == self.admin.is_library_manager(other_library)
-        assert True == self.admin.is_librarian(self._default_library)
-        assert True == self.admin.is_librarian(other_library)
-
-    def test_can_see_collection(self):
-        # This collection is only visible to system admins since it has no libraries.
-        c1 = self._collection()
-
-        # This collection is visible to libraries of its library.
-        c2 = self._collection()
-        c2.libraries += [self._default_library]
-
-        # The admin has no roles yet.
-        assert False == self.admin.can_see_collection(c1);
-        assert False == self.admin.can_see_collection(c2);
-
-        self.admin.add_role(AdminRole.SYSTEM_ADMIN)
-        assert True == self.admin.can_see_collection(c1)
-        assert True == self.admin.can_see_collection(c2)
-
-        self.admin.remove_role(AdminRole.SYSTEM_ADMIN)
-        self.admin.add_role(AdminRole.SITEWIDE_LIBRARY_MANAGER)
-        assert False == self.admin.can_see_collection(c1);
-        assert True == self.admin.can_see_collection(c2);
-
-        self.admin.remove_role(AdminRole.SITEWIDE_LIBRARY_MANAGER)
-        self.admin.add_role(AdminRole.SITEWIDE_LIBRARIAN)
-        assert False == self.admin.can_see_collection(c1);
-        assert True == self.admin.can_see_collection(c2);
-
-        self.admin.remove_role(AdminRole.SITEWIDE_LIBRARIAN)
-        self.admin.add_role(AdminRole.LIBRARY_MANAGER, self._default_library)
-        assert False == self.admin.can_see_collection(c1);
-        assert True == self.admin.can_see_collection(c2);
-
-        self.admin.remove_role(AdminRole.LIBRARY_MANAGER, self._default_library)
-        self.admin.add_role(AdminRole.LIBRARIAN, self._default_library)
-        assert False == self.admin.can_see_collection(c1);
-        assert True == self.admin.can_see_collection(c2);
-
-        self.admin.remove_role(AdminRole.LIBRARIAN, self._default_library)
-        assert False == self.admin.can_see_collection(c1);
-        assert False == self.admin.can_see_collection(c2);
+
+def test_password_hashed(db_session, create_admin_user):
+    # GIVEN: An Admin user
```

Comment:
It'd probably be good if these were actual docstrings. [PEP 257](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) says that's "a string literal that occurs as the first statement in a module, function, class, or method definition," so you have to quote it instead of using single-line comment syntax for it to be an actual docstring. I'm not sure there's an _immediately_ obvious benefit to docstring vs. comment, but the argument for using them is that real docstrings end up in `whatever.__doc__` and can be introspected by various pieces of pre-process and run-time tooling like the [`help()`](https://docs.python.org/3/library/functions.html#help) built-in, IDE parsers, etc. I think inline comments that aren't string literals are just discarded during compilation, so you can't read them at runtime unless you bring in something like [`inspect.getsource()`](https://docs.python.org/3/library/inspect.html#inspect.getsource), and even that module expects there to be docstrings for stuff like `inspect.cleandoc()`.